.circleci/config.yml

```diff
@@ -46,3 +56,12 @@ workflows:
 
       - lint_ruby:
           requires: [carwow/start]
+
```

Comment:
I am not sure if this will work, if it doesn't I will remove it. Once I have tested it I will update the README to reflect how to publish the gem.